src/main/webapp/assets/js/analytics.js

```diff
@@ -0,0 +1,80 @@
+$(document).ready(function(){
+	var term='term';
+	getTermsInfo();
+    
+    function getTermsInfo(){
+    	var url="analytics?action=getEnrollmentTerms";
+    	var termRequest = $.ajax({
+			url: url,
+			type: "GET",			
+			dataType: "json"
+		});
+    	termRequest.done(function(data) {
+    		var terms=data.enrollment_terms;
+    		var newSelect=document.createElement('select');
```

Comment:
Might be better to have an initial empty select that gets populated with option children based on the data. Also this select, regardless who renders it, needs to have a <label> whose "for" attribute matches the ID of the select.